_networktables_types.py

```diff
@@ -0,0 +1,11 @@
+from typing import Sequence, Union
```

Comment:
wtf is this extra file for?

Reply:
Breaking what would otherwise be a cyclic import. I'd just put this at `networktables/types.py`, but… yeah.